argo-cd-apps/base/member/infra-deployments/segment-bridge/segment-bridge-member.yaml

```diff
@@ -19,18 +19,22 @@ spec:
                   values.clusterDir: stone-stg-m01
                 - nameNormalized: stone-stg-rh01
                   values.clusterDir: stone-stg-rh01
+                - nameNormalized: stone-prd-m01
+                  values.clusterDir: stone-stg-m01
+                - nameNormalized: stone-prd-rh01
+                  values.clusterDir: stone-stg-rh01
   template:
     metadata:
       name: segment-bridge-member-{{nameNormalized}}
     spec:
       project: default
       source:
-        path: '{{values.sourceRoot}}/{{values.environment}}/{{values.clusterDir}}'
+        path: "{{values.sourceRoot}}/{{values.environment}}/{{values.clusterDir}}"
         repoURL: https://github.com/redhat-appstudio/infra-deployments.git
         targetRevision: main
       destination:
         namespace: segment-bridge
-        server: '{{server}}'
+        server: "{{server}}"
```

Comment:
do you want to keep those unrelated changes? i.e. changing the single quote to double. I am ok with them

Reply:
usually I'll remove it to shrink unnecessary updates out of this change (linter artifact), but since most of the review (tests) steps have passed successfully, I don't mind leaving it in.